fhir-bulkimportexport-webapp/liberty-config/server.xml

```diff
@@ -0,0 +1,203 @@
+<server description="fhir-server">
```

Comment:
use src/main/liberty/config it's the pattern the OpenLiberty team uses, and we can do some nice automation on top

Reply:
I was following the same pattern that we use in fhir-server project, but I agree that the new pattern is better, will change.

Reply:
done